guacamole/src/main/frontend/src/app/client/directives/guacClientNotification.js

```diff
@@ -65,26 +66,6 @@ angular.module('client').directive('guacClientNotification', [function guacClien
          */
         $scope.status = false;
 
-        /**
-         * All client error codes handled and passed off for translation. Any error
-         * code not present in this list will be represented by the "DEFAULT"
-         * translation.
-         */
-        const CLIENT_ERRORS = {
-            0x0201: true,
-            0x0202: true,
-            0x0203: true,
-            0x0207: true,
-            0x0208: true,
-            0x0209: true,
-            0x020A: true,
-            0x020B: true,
-            0x0301: true,
-            0x0303: true,
-            0x0308: true,
-            0x031D: true
-        };
```

Comment:
Are these error codes not needed at all anymore? I see that they're removed here and in the `guacFileTransfer.js` file, but do they not need to be defined at all?

Reply:
Yeah, we don't need these anymore - they were just a hardcoded list of error codes for which translations were assumed to exist. 

Now it's checking the available translations, using a status-code-specific one if it exists, and otherwise just going with the default.

They weren't used for anything else.